framework/doc/static_bundles.rst

```diff
@@ -5,13 +5,16 @@ Static Bundles
 
 The normal and most flexible way to add a CppMicroServices bundle to
 an application is to compile it into a shared library that is later
-:any:`installed <cppmicroservices::BundleContext::InstallBundles>` in the application. 
+installed programmatically using the API 
+:any:`installed <cppmicroservices::BundleContext::InstallBundles>` at
+runtime.
```

Comment:
I suggest to write "using the :any:`BundleContext::InstallBundles() <cppmicroservices::BundleContext::InstallBundles>` function at runtime" instead.